xapian-letor/tests/api_letor.cc

```diff
@@ -507,42 +524,36 @@ DEFINE_TESTCASE(listnet_ranker_three_correct, generated)
     ranker.train_model(training_data);
     Xapian::docid doc1 = *mymset[0];
     Xapian::docid doc2 = *mymset[1];
-    // Xapian::docid doc3 = *mymset[2];
     ranker.rank(mymset);
     TEST_EQUAL(doc1, *mymset[1]);
     TEST_EQUAL(doc2, *mymset[0]);
-    // TEST_EQUAL(doc3, *mymset[2]);
     mymset = enquire.get_mset(0, 10);
     ranker.train_model(training_data, "ListNet_Ranker");
     ranker.rank(mymset, "ListNet_Ranker");
     TEST_EQUAL(doc1, *mymset[1]);
     TEST_EQUAL(doc2, *mymset[0]);
-    // TEST_EQUAL(doc3, *mymset[2]);
     TEST_EXCEPTION(Xapian::LetorInternalError,
 		   ranker.score(query, qrel, "ListNet_Ranker",
-				"scorer_output3.txt", 10, ""));
+				"scorer_output.txt", 10, ""));
```

Comment:
It seems you've changed all the testcases to use the same filename - `scorer_output.txt`.  Before they used separate filenames, which meant they could be run concurrently.  Now if they're run concurrently they'll overwrite each others output.  We really don't want to create obstacles to parallelising the testsuite.

Reply:
Oh, or are these written into the path set by `ranker.set_database_path(db_path)`?

Reply:
They're just to test exception, so they dont matter right?